src/ray/core_worker/core_worker.cc

```diff
@@ -619,7 +619,6 @@ void CoreWorker::Disconnect(
     const std::string &exit_detail,
     const std::shared_ptr<LocalMemoryBuffer> &creation_task_exception_pb_bytes) {
   // Force stats export before exiting the worker.
-  opencensus::stats::StatsExporter::ExportNow();
```

Comment:
I moved it to stats::Shutdown (so it is more generally applied to all components)